lcUILua/ui/commandline.lua

```diff
@@ -47,12 +52,17 @@ end
 
 --Create the command line and add it to the main window
 function add_commandline(mainWindow, id)
-    local cliCommand = lc.CliCommand(mainWindow)
+    cliCommand = lc.CliCommand(mainWindow)
```

Comment:
This is not a good idea of making `cliCommand` global, as it won't support multiple windows.You can access the CliCommand widget globally using `cliCommands[window_id]`.